<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Safety_Param_Dialog</class>
 <widget class="QDialog" name="Safety_Param_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>安全参数</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="4" column="2">
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="2">
      <widget class="QPushButton" name="cancle_pushButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="apply_pushButton">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_48">
     <property name="text">
      <string>标签名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page_ai">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QSpinBox" name="ai_simple_interval_spinBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据互检的容忍误差值的其中一个来源: 满量程(24V)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>互检满量程占比</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ai_full_spinbox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据互检的容忍误差值的其中一个来源: 满量程(24V)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="ai_actual_spinbox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据互检的容忍误差值的其中一个来源: AI实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据互检的容忍误差值的其中一个来源: AI实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>互检实时数据占比</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;模拟输入(AI)数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>模拟输入采样间隔</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_di">
      <layout class="QGridLayout" name="gridLayout_21"/>
     </widget>
     <widget class="QWidget" name="page_relay">
      <layout class="QGridLayout" name="gridLayout_24"/>
     </widget>
     <widget class="QWidget" name="page_mos">
      <layout class="QGridLayout" name="gridLayout_28"/>
     </widget>
     <widget class="QWidget" name="page_encode">
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_20">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{border: none solid gray;font:bold 11px;}
QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top left; }</string>
         </property>
         <property name="title">
          <string>同步互检:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_64">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>编码器采样间隔</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="qep_simple_interval_spinBox">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_65">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据互检的容忍误差值的其中一个来源: 满量程(200k)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>互检满量程占比</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="qep_full_spinBox">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据互检的容忍误差值的其中一个来源: 满量程(200k)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_66">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据互检的容忍误差值的其中一个来源: 编码器实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>互检实时数据占比</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="qep_actual_spinBox">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;编码器数据互检的容忍误差值的其中一个来源: 编码器实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_encode_param">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{border: none solid gray;font:bold 11px;}
QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top left; }</string>
         </property>
         <property name="title">
          <string>编码器参数:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="text">
             <string>个脉冲</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_pulse_num">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="text">
             <string>输出轴每转一圈，编码器输出</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>输出轴每转一圈，机器运动</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_distance">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>groupBox_encode_param</zorder>
      <zorder>groupBox_20</zorder>
      <zorder>line</zorder>
      <zorder>horizontalSpacer_3</zorder>
     </widget>
     <widget class="QWidget" name="page_pi">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_71">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>脉冲输入采样间隔</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_74">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据互检的容忍误差值的其中一个来源: PI实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>互检实时数据占比</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="pi_actual_spinBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据互检的容忍误差值的其中一个来源: PI实时数据*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_73">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据互检的容忍误差值的其中一个来源: 满量程(200k)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>互检满量程占比</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="pi_full_spinBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据互检的容忍误差值的其中一个来源: 满量程(200k)*此百分比&lt;/p&gt;&lt;p&gt;单位:%&lt;/p&gt;&lt;p&gt;阈值：0-100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="pi_simple_interval_spinBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;脉冲输入(PI)数据的采样间隔&lt;/p&gt;&lt;p&gt;单位:ms&lt;/p&gt;&lt;p&gt;阈值：10-200&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
